barretenberg/cpp/src/barretenberg/dsl/acir_format/honk_recursion_constraint.test.cpp

```diff
@@ -127,7 +127,29 @@ class AcirHonkRecursionConstraint : public ::testing::Test {
         };
         auto builder =
             create_circuit(constraint_system, /*recursive*/ true, /*size_hint*/ 0, witness, /*honk recursion*/ true);
-
+        if constexpr (HasIPAAccumulator<Flavor>) {
```

Comment:
adds the IPA claim and proof to the inner circuit when we want to do rollup stuff